test/unit/reductions/grid_search/test_grid_generator.py

```diff
@@ -34,6 +34,31 @@ def test_grid_generator_demographic_parity(grid_size, grid_limit):
     assert np.isclose(expected_grid.values, grid.values).all()
 
 
+@pytest.mark.parametrize("grid_size", [5, 6, 10, 11, 100, 101, 102])
+@pytest.mark.parametrize("grid_limit", [1.0, 2.0, 4.0, 10.0])
+def test_grid_generator_demographic_parity_with_center(grid_size, grid_limit):
+    disparity_moment = DemographicParity()
+    events = [_ALL]
+
+    _index = pd.MultiIndex.from_product([['+', '-'], events, [0, 1]],
+                                        names=[_SIGN, _EVENT, _GROUP_ID])
+    add_val = 0.02
+    grid_center = pd.Series([0, 0, 0, add_val], index=_index)
```

Comment:
Other than the grid_center this looks identical to the test case above. Can we instead parametrize on a few different center values
```python
@pytest.mark.parametrize("grid_center",[
    pd.Series([0, 0, 0, 0],
    pd.Series([0, 0, 0, 1],
    pd.Series([0, 1, 0, 0],
    pd.Series([0, 0, 0, 0.25],
    pd.Series([0, 0.25, 0, 0]
])
```
?

Reply:
I suppose the same applies below, although the vector is twice as long and we'd need to consider every other component as non-zero in at least some test.

Reply:
Sure, I'll do that.

Reply:
In the changes I only see new cases, but no consolidation with the existing ones. Were there any issues or do you just need a little more time? Either way is fine, just trying to see whether you need help unblocking this.